ambry-api/src/main/java/com.github.ambry/network/NetworkRequestSend.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.network;
+
+/**
+ * An interface to update send time for a network request
+ */
+public interface NetworkRequestSend {
```

Comment:
You don't even need to do (1) - but I would like that anyway because I don't see the value of `MetricsHistogram`.